### PR TITLE
Add `bits` option

### PR DESCRIPTION
**Pull request by montyanderson**
Switch units based on boolean 'bits' flag.

### CODE REVIEW

**Comment by sindresorhus** [comment]
What’s the use-case? Would be good to include that in the docs for the option.

**Comment by super3** [comment]
@sindresorhus Download/upload speed is usually represented in bits/s vs bytes/s. We are using this for logging on [YaCDN](https://github.com/ovsoinc/yacdn.org):

```
{ n: 14291272,
4|yacdn-5  |   url:
4|yacdn-5  |    'http://meowbaari.com/wp-content/uploads/2016/06/1464933654_cat_sleep.png',
4|yacdn-5  |   referer: undefined,
4|yacdn-5  |   size: '3.05 kB',
4|yacdn-5  |   time: '2101ms',
4|yacdn-5  |   *speed: '1.45 kbps* }
```

File: index.d.ts [hunk 1]
```diff
@@ -35,13 +43,21 @@ prettyBytes(1337);
...
 prettyBytes(42, {signed: true});
 //=> '+42 B'
 
 // Localized output using German locale
 prettyBytes(1337, {locale: 'de'});
 //=> '1,34 kB'
+
+// Display with units of bits
+prettyBytes(1337, {bits: true});
+// => '1,34 kb'
```
Comment (sindresorhus) [inline index.d.ts]: This is not correct

File: index.d.ts [hunk 2]
```diff
@@ -35,13 +43,21 @@ prettyBytes(1337);
 prettyBytes(100);
 //=> '100 B'
 
+// Display with units of bits
+prettyBytes(1337, {bits: true});
+// => '1.34 kbit'
```
Comment (sindresorhus) [inline index.d.ts]: Can you put this together with the option instead? At line 24.

File: index.js [hunk 3]
```diff
@@ -35,9 +47,10 @@ module.exports = (number, options) => {
 	}
 
 	options = Object.assign({}, options);
```
Comment (sindresorhus) [inline index.js]:
```suggestion
	options = Object.assign({bits: false}, options);
```

File: index.js [hunk 4]
```diff
@@ -35,9 +47,10 @@ module.exports = (number, options) => {
...
+	const UNITS = options.bits === true ? BIT_UNITS : BYTE_UNITS;
```
Comment (sindresorhus) [inline index.js]:
```suggestion
	const UNITS = options.bits ? BIT_UNITS : BYTE_UNITS;
```

File: readme.md [hunk 5]
```diff
@@ -57,6 +61,12 @@ Default: `false`
 
 Include plus sign for positive numbers. If the difference is exactly zero a space character will be prepended instead for better alignment.
 
+##### bits
+
+Type: `boolean`<br>
+Default: `false`
+
+Treat input as bits and use proper units respectively.
```
Comment (sindresorhus) [inline readme.md]:
```suggestion
Treat input as [bits](https://en.wikipedia.org/wiki/Bit) and use proper units respectively.
```

**Comment by sindresorhus** [comment]
> Download/upload speed is usually represented in bits/s vs bytes/s. We are using this for logging on YaCDN:

👍 Makes sense. Would be good to include the use-case about speed in the docs for the option.

**Comment by super3** [comment]
@sindresorhus Are the changes satisfactory?

**Pull request merged**